blivet/populator/helpers/__init__.py

```diff
@@ -1,4 +1,5 @@
 import inspect as _inspect
+import six as _six
```

Comment:
Still don't see the point. It only makes difference for ``*`` imports, doesn't it? Maybe we should define ``__all__`` instead?